R/effectivemass.R

```diff
@@ -148,7 +148,7 @@ fit.effectivemass <- function(cf, t1, t2, useCov=FALSE, replace.na=TRUE, boot.fi
     stop("t1 and t2 must be specified! Aborting...!\n")
   }
   tmax <- cf$Time/2
-  if(!is.null(cf$cf$symmetrised)) {
+  if( "symmetrised" %in% names(cf) ) {
```

Comment:
Before it was `cf$cf$symmetrised`, now it is `names(cf)`. I would think that we need `names(cf$cf)` then.

Reply:
Yup.

Reply:
Even better would be to not name this object "cf" :)